SimPEG/Examples/DC_Forward_PseudoSection.py

```diff
@@ -0,0 +1,215 @@
+        P2 = mesh.getInterpolationMat(rxloc_N, 'CC')
+
+        # Compute the potential difference
+        dtemp = (P1*phi - P2*phi)*np.pi
+
+        data.append( dtemp )
+        print ('\rTransmitter {0} of {1} -> Time:{2} sec'.format(ii,len(Tx),time.time()- start_time),)
+
+    print ('Transmitter {0} of {1}'.format(ii,len(Tx)))
+    print ('Forward completed')
+
+    # Let's just convert the 3D format into 2D (distance along line) and plot
+    survey2D = convertObs_DC3D_to_2D(survey, np.ones(survey.nSrc) , 'Xloc')
+    survey2D.dobs =np.hstack(data)
+
+    if plotIt:
+        import matplotlib.pyplot as plt
+        fig = plt.figure(figsize=(7,7))
+        ax = plt.subplot(2,1,1, aspect='equal')
+        # Plot the location of the spheres for reference
+        circle1=plt.Circle((loc[0,0], loc[2,0]), radi[0], color='w', fill=False, lw=3)
+        circle2=plt.Circle((loc[0,1], loc[2,1]), radi[1], color='k', fill=False, lw=3)
+        ax.add_artist(circle1)
+        ax.add_artist(circle2)
+
+        dat = mesh.plotSlice(np.log10(model), ax = ax, normal = 'Y',
+                             ind = indy,grid=True, clim = np.log10([sig.min(),sig.max()]))
+
+        ax.set_title('3-D model')
+        plt.gca().set_aspect('equal', adjustable='box')
+        plt.scatter(Tx[0].loc[0][0],Tx[0].loc[0][2],s=40,c='g', marker='v')
+        plt.scatter(Rx[0].locs[0][:,0],Rx[0].locs[0][:,1],s=40,c='y')
+        plt.xlim([-xlim,xlim])
+        plt.ylim([-zlim,mesh.vectorNz[-1]+dx])
+
+
+        pos =  ax.get_position()
+        ax.set_position([pos.x0 , pos.y0 + 0.025 ,  pos.width, pos.height])
+        pos =  ax.get_position()
+        cbarax = fig.add_axes([pos.x0 , pos.y0 + 0.025 ,  pos.width, pos.height * 0.04])  ## the parameters are the specified position you set
+        cb = fig.colorbar(dat[0],cax=cbarax, orientation="horizontal",
+                          ax = ax, ticks=np.linspace(np.log10(sig.min()),
+                         np.log10(sig.max()), 3), format="$10^{%.1f}$")
+        cb.set_label("Conductivity (S/m)",size=12)
+        cb.ax.tick_params(labelsize=12)
+
+        # Second plot for the predicted apparent resistivity data
+        ax2 = plt.subplot(2,1,2, aspect='equal')
+
+        # Plot the location of the spheres for reference
+        circle1=plt.Circle((loc[0,0], loc[2,0]), radi[0], color='w', fill=False, lw=3)
+        circle2=plt.Circle((loc[0,1], loc[2,1]), radi[1], color='k', fill=False, lw=3)
+        ax2.add_artist(circle1)
+        ax2.add_artist(circle2)
+
+        # Add the speudo section
+        dat = plot_pseudoSection(survey2D, ax2, surveyType=surveyType, dataType=unitType)        # plt.scatter(Tx2d[0][:],Tx[0][2,:],s=40,c='g', marker='v')
+        # plt.scatter(Rx2d[0][:],Rx[0][:,2::3],s=40,c='y')
+        # plt.plot(np.r_[Tx2d[0][0],Rx2d[-1][-1,-1]],np.ones(2)*mesh.vectorNz[-1], color='k')
+        ax2.set_title('Apparent Conductivity data')
+
+        plt.ylim([-zlim,mesh.vectorNz[-1]+dx])
+        plt.show()
```

Comment:
this piece needs to be outside of the main function. move it down below the `run()`